_health-care/_js/components/Nav.jsx

```diff
@@ -10,85 +11,97 @@ import React from 'react';
 class Nav extends React.Component {
   render() {
     const subnavStyles = 'step one wow fadeIn animated';
+    const completedSections = this.props.data.completedSections;
+    const currentUrl = this.props.currentUrl;
     // TODO(akainic): change this check once the alias for introduction has been changed
     return (
       <ol className="process hca-process">
-        <li className={`one ${subnavStyles} ${this.props.currentUrl.startsWith('/introduction') ? ' section-current' : ''}`}>
+        <li className={`one ${subnavStyles} ${currentUrl.startsWith('/introduction') ? ' section-current' : ''}
+          ${(completedSections['/introduction'] === true) ? ' section-complete' : ''}`}>
```

Comment:
I wonder if we can DRY some of this up by extracting it into a function that takes the url as a parameter? For example:
```javascript
function determineStyles(path)
  let classes = '';
  if (currentUrl.startsWith(path)) {
    classes += ' section-current';
  }
  if (completedSections[path] === true) {
    classes += ' section-complete';
  }
  return classes;
}
```
And then we can just call `determineStyles('/introduction')` here and it will return the necessary class names? Not positive that will work exactly, but something like that.